### PR TITLE
Update crons.md

### PR DESCRIPTION
**Pull request by launay12u**
Adding a more concrete example for using cron with environment variables

### CODE REVIEW

**Comment by hsablonniere** [comment]
I would mention the fact tha `$ROOT` is the only variable accessible to cron jobs (before sourcing the others). I got confused with this recently.

**Comment by hsablonniere** [comment]
Out of curiosity why did you use ```kaskell for JSON code example?

File: commons/tools/crons.md [hunk 1]
```diff
@@ -50,14 +50,42 @@ Environment variables are not available in scripts / executables ran from crons.
 
 To do so, your cron should call a shell script which will inject the environment and in turn run your command:
 
-```bash
+In `clevercloud/cron.json`:
+
+```haskell
```
Comment (urcadox) [inline commons/tools/crons.md]: Why indeed?

File: commons/tools/crons.md [hunk 2]
```diff
@@ -50,14 +50,42 @@ Environment variables are not available in scripts / executables ran from crons.
...
+And in `script.sh`:
+
+```bash 
 #! /usr/bin/env bash
 
 source /home/bas/applicationrc
 
 /usr/bin/php $APP_HOME/cron.php
 ```
+Now you can call environment variables as you want in `cron.php`.
```
Comment (urcadox) [inline commons/tools/crons.md]: "Now you have access to environment variables in..." sounds better to me

File: commons/tools/crons.md [hunk 3]
```diff
@@ -50,14 +50,42 @@ Environment variables are not available in scripts / executables ran from crons.
...
+```bash 
 #! /usr/bin/env bash
 
 source /home/bas/applicationrc
 
 /usr/bin/php $APP_HOME/cron.php
 ```
+Now you can call environment variables as you want in `cron.php`.   
+
+If you need to inject environnement variables in more than one cron you pass the cron in arguments on the bash script:
```
Comment (urcadox) [inline commons/tools/crons.md]: s/one cron you pass the cron in arguments on the bash script/one cron; you can pass the file in an argument of the bash script/

File: commons/tools/crons.md [hunk 4]
```diff
@@ -50,14 +50,42 @@ Environment variables are not available in scripts / executables ran from crons.
...
 source /home/bas/applicationrc
 
 /usr/bin/php $APP_HOME/cron.php
 ```
+Now you can call environment variables as you want in `cron.php`.   
+
+If you need to inject environnement variables in more than one cron you pass the cron in arguments on the bash script:  
+In `clevercloud/cron.json`:
 
+```haskell
```
Comment (urcadox) [inline commons/tools/crons.md]: same thing as above

**Comment by launay12u** [comment]
I just made the changes you noted above, everything is good?

**Comment by urcadox** [comment]
Oh sorry for the abrupt review, I thought a colleague made that pull request. Let me review that :)

**Comment by urcadox** [comment]
Alright, that's perfect. Thanks!

**Comment by launay12u** [comment]
Oh no problem, Thanks !

**Comment by hsablonniere** [comment]
@launay12u Thanks :heart_eyes:

**Comment by launay12u** [comment]
I'm glad to have been helpful.

**Pull request merged**